style.css

```diff
@@ -391,7 +402,145 @@ hr {
   margin-left: 32px;
 }
 
-@media only screen and (max-width: 768px) {
+/* validate */
+
+.error-message {
+  color: red;
+  font-size: 17px;
+  align-self: center;
+  padding-bottom: 5px;
+}
+
+.error {
+  margin-bottom: 15px;
+  height: 48px;
+  border-radius: 8px;
+  font-size: 17px;
+  border: 1px solid red;
+  padding: 16px;
+  color: #172b4d;
+}
+
+/* popup-window */
+
+.pop-main {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1;
+  top: 0;
+  left: 0;
+  visibility: hidden;
+}
+
+.pop-container {
+  display: flex;
+  margin: 20px 12px;
+  flex-flow: wrap column;
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 16px;
+}
```

Comment:
- [ ] GOOD JOB on using the data present in a javascript array of objects 👏  👏  and also on making designing your mobile version popup window But one thing is left. One of the requirements says that the popup is responsive, it fits mobile and desktop sizes properly. Kindly check the screenshot here down 👇  of your popup design and make changes by following the template design. 
### your design
![Screen Shot 2021-12-18 at 14 35 05](https://user-images.githubusercontent.com/65068771/146641510-81a20c1d-3970-459a-bd9c-c0fd39505eac.png)
### The template 
![Screen Shot 2021-12-18 at 14 45 54](https://user-images.githubusercontent.com/65068771/146641604-3bb88474-a92d-4074-bea4-ea4bda88bb6b.png)